app/bundles/CoreBundle/Controller/BuilderControllerTrait.php

```diff
@@ -31,7 +31,7 @@ protected function getAssetsForBuilder()
             ->addScriptDeclaration("var mauticAjaxUrl     = '".$routerHelper->generate('mautic_core_ajax')."';")
             ->addScriptDeclaration("var mauticBaseUrl     = '".$routerHelper->generate('mautic_base_index')."';")
             ->addScriptDeclaration("var mauticAssetPrefix = '".$assetsHelper->getAssetPrefix(true)."';")
-            ->addScriptDeclaration("var mauticLang         = '".$this->get('templating.helper.translator')->getJsLang()."';")
+            ->addScriptDeclaration("var mauticLang        = '".str_replace("'", "\\'", $this->get('templating.helper.translator')->getJsLang())."';")
```

Comment:
Let's do this instead:

```
    "var mauticLang        = ".$view['translator']->getJsLang().";"
```

That's how we are doing it here https://github.com/mautic/mautic/blob/c90f59d5b56608190fac791f6c008f40db29c733/app/bundles/CoreBundle/Views/Default/script.html.php#L25-L25 so that it's a json object.